helix-term/src/keymap.rs

```diff
@@ -578,11 +578,13 @@ impl Default for Keymaps {
             "C-u" => half_page_up,
             "C-d" => half_page_down,
 
+            // TODO extract the common binding with `Space w`
```

Comment:
We can't really do that easily so I would remove the TODO

Reply:
done